corehq/apps/settings/views.py

```diff
@@ -385,6 +386,19 @@ def dispatch(self, request, *args, **kwargs):
         return super(TwoFactorPhoneSetupView, self).dispatch(request, *args, **kwargs)
 
 
+class TwoFactorPhoneDeleteView(BaseMyAccountView, PhoneDeleteView):
+    success_url = "/account/two_factor/"
+
+    def get_success_url(self):
+        messages.add_message(self.request, messages.SUCCESS, "Phone number added!")
```

Comment:
This message needs to be translated

Reply:
Done: https://github.com/dimagi/commcare-hq/pull/19577/commits/d0457f4eb3c621b458100b9b98c4743c5ad019a3